Description: Fix some profiles.
 .
 firejail (0.9.50-3parrot0) testing; urgency=medium
 .
   * Non-maintainer upload.
   * Fix thunderbird rules.
   * Fix display rules.
   * Fix libreoffice rules.
Author: Lorenzo "Palinuro" Faletra <palinuro@parrotsec.org>
Last-Update: 2017-10-30

--- firejail-0.9.50.orig/etc/atril.profile
+++ firejail-0.9.50/etc/atril.profile
@@ -7,6 +7,7 @@ include /etc/firejail/globals.local
 
 noblacklist ~/.config/atril
 noblacklist ~/.local/share
+noblacklist /usr/share/doc
 
 include /etc/firejail/disable-common.inc
 include /etc/firejail/disable-devel.inc
--- firejail-0.9.50.orig/etc/display.profile
+++ firejail-0.9.50/etc/display.profile
@@ -22,9 +22,9 @@ notv
 protocol unix
 seccomp
 shell none
-x11 xorg
+#x11 xorg
 
-private-bin display
+#private-bin display
 private-dev
-private-etc none
+#private-etc none
 private-tmp
--- firejail-0.9.50.orig/etc/libreoffice.profile
+++ firejail-0.9.50/etc/libreoffice.profile
@@ -10,7 +10,7 @@ noblacklist /usr/local/sbin
 noblacklist ~/.config/libreoffice
 
 include /etc/firejail/disable-common.inc
-include /etc/firejail/disable-devel.inc
+#include /etc/firejail/disable-devel.inc
 include /etc/firejail/disable-passwdmgr.inc
 include /etc/firejail/disable-programs.inc
 
@@ -28,5 +28,5 @@ tracelog
 
 private-dev
 
-noexec ${HOME}
+#noexec ${HOME}
 noexec /tmp
--- firejail-0.9.50.orig/etc/thunderbird.profile
+++ firejail-0.9.50/etc/thunderbird.profile
@@ -10,16 +10,13 @@ include /etc/firejail/globals.local
 
 noblacklist ~/.cache/thunderbird
 noblacklist ~/.gnupg
-noblacklist ~/.icedove
 noblacklist ~/.thunderbird
 
 mkdir ~/.cache/thunderbird
 mkdir ~/.gnupg
-mkdir ~/.icedove
 mkdir ~/.thunderbird
 whitelist ~/.cache/thunderbird
 whitelist ~/.gnupg
-whitelist ~/.icedove
 whitelist ~/.local/share/applications
 whitelist ~/.thunderbird
 include /etc/firejail/whitelist-common.inc
--- firejail-0.9.50.orig/etc/firefox.profile
+++ firejail-0.9.50/etc/firefox.profile
@@ -29,6 +29,7 @@ mkdir ~/.cache/mozilla/firefox
 mkdir ~/.mozilla
 mkdir ~/.pki
 whitelist ${DOWNLOADS}
+whitelist ~/Desktop
 whitelist ~/.cache/gnome-mplayer/plugin
 whitelist ~/.cache/mozilla/firefox
 whitelist ~/.config/gnome-mplayer
